### PR TITLE
Feat/create list activity

### PR DESCRIPTION
**Pull request by anangkur**
changes:
- setup jetpack compose.
- create splash activity.
- create list activity.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -22,15 +22,50 @@ android {
...
+    // androidx
     implementation 'androidx.core:core-ktx:1.3.0'
     implementation 'androidx.appcompat:appcompat:1.1.0'
-    testImplementation 'junit:junit:4.12'
+
+    // compose
+    implementation 'androidx.ui:ui-core:0.1.0-dev14'
+    implementation 'androidx.ui:ui-tooling:0.1.0-dev14'
+    implementation 'androidx.ui:ui-layout:0.1.0-dev14'
+    implementation 'androidx.ui:ui-material:0.1.0-dev14'
```
Comment (mochadwi) [inline app/build.gradle:60]: this is the jetpack compose dependencies? @anangkur

File: app/src/main/java/com/anangkur/jetpackcomposeplayground/ui/list/ListActivity.kt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                    ListItem(title = "Test 1", desc = "Desc Test 1", image = ""),
+                    ListItem(title = "Test 2", desc = "Desc Test 2", image = ""),
+                    ListItem(title = "Test 3", desc = "Desc Test 3", image = ""),
+                    ListItem(title = "Test 4", desc = "Desc Test 4", image = ""),
+                    ListItem(title = "Test 5", desc = "Desc Test 5", image = "")
+                ),
+                onClick = { listItem -> this.onClickItem(listItem) }
+            )
+        }
+    }
```
Comment (mochadwi) [inline app/src/main/java/com/anangkur/jetpackcomposeplayground/ui/list/ListActivity.kt:36]: Naisuuuu.

File: app/src/main/java/com/anangkur/jetpackcomposeplayground/ui/list/ListItemScreen.kt [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import androidx.ui.foundation.clickable
+import androidx.ui.layout.*
+import androidx.ui.material.Card
+import androidx.ui.text.font.FontWeight
+import androidx.ui.tooling.preview.Preview
+import androidx.ui.unit.dp
+import androidx.ui.unit.sp
+import com.anangkur.jetpackcomposeplayground.model.ListItem
+
+@Composable
```
Comment (mochadwi) [inline app/src/main/java/com/anangkur/jetpackcomposeplayground/ui/list/ListItemScreen.kt:15]: This is the most fun part, a reusable component :D @anangkur

File: app/build.gradle (hunk 1 above)
Comment (anangkur) [reply app/build.gradle:60]: yup mas, I refer to this tutorial
https://developer.android.com/jetpack/compose/setup

File: app/src/main/java/com/anangkur/jetpackcomposeplayground/ui/list/ListItemScreen.kt (hunk 3 above)
Comment (anangkur) [reply app/src/main/java/com/anangkur/jetpackcomposeplayground/ui/list/ListItemScreen.kt:15]: agree mas, all components are reusable.

**Pull request merged**